### ISSUE TITLE
Custom repo and Group images missing

### ISSUE THREAD

**Issue by lululujojo123** [labnex/LabNex#41]
**I confirm that I have searched the issue tracker**
Before opening an issue, take a few minutes to search the issue tracker (both open and closed issues) for the problem you are facing. This will reduce the need for issue triage and save time.

 - [x] I confirm that I have searched the issue tracker

**Describe the bug**
The custom repo images and group images seem not to load on self hosted instance of Gitlab. Either the progress cycler keeps spinning or the General image with first letter is appearing.

**To Reproduce**
Not really sure how to reproduce. May a docker image with custom Gitlab instance. Afterwards setting a custom repo image and try to load the group view or repo view.

**Expected behavior**
The image should appear in the image area instead of the progress indicator or the General letter.

**Logs**
Seems like no logs available for this error. May I have some problems with logcat.

**Screenshots**
![Screenshot_20241231-135337.png](https://github.com/user-attachments/assets/57454869-60a6-47e1-ab27-1238888409c9)

![Screenshot_20241231-135241.png](https://github.com/user-attachments/assets/b52fabf0-a5a8-415b-bcc6-948b65f850f5)

**Versions (please complete the following information)**
 - LabNex:  2.0.0 - custom build with OffsetDateTime
 - Android:  12
 - GitLab:  n/a

**GitLab instance**
 - [ ] gitlab.com
 - [ ] framagit.org
 - [x] Self-hosted instance
 - [ ] Other service (Name/URL: )

**Mode of installation**
 - [ ] Google Play Store
 - [ ] F-droid
 - [ ] APK
 - [x] Built from source
 - [ ] OpenAPK
 - [ ] Other stores (Name: )

**Additional context**
Doesn't effect further usage of the app.

**Comment by mmarif4u** [comment]
Does this happen every time you load the app, or just once on that particular screen?

**Comment by lululujojo123** [comment]
> Does this happen every time you load the app, or just once on that particular screen?

Yes this happens every time. Doesn't change at all. Those are private projects/groups as well? May it also need an authorization header to be accessed?
